docs/install/installation.rst

```diff
@@ -17,31 +19,36 @@ Prerequisites
 
 - A C++ 17 compiler such as GCC 7 or Clang 5.
 
-- The AMD ROCm software stack. See the `ROCm installation instructions <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/index.html>`_.
+- The AMD ROCm software stack. See the `ROCm installation instructions <rocm-install-on-linux:index>`.
+
+- Install the required packages according to the OS:
 
-- Packages as per the OS.
+.. tab-set::
 
-  - For Ubuntu 18.04 and Ubuntu 20.04:
+  .. tab-item:: Ubuntu 18.04 and Ubuntu 20.04
```

Comment:
Those could be upgraded. I think 20.04 is "officially" unsupported (or close to be),  18.04 is probably not supported anymore.  Currently supported LTS releases are 22.04 and 24.04.

That being said, the dependencies we have are quite stable and should be the same across all available ubuntu versions.

Reply:
Install commands and package names have not changed.  The only thing is that the distro version might end-up out of sync with https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html.

Maybe it would be easier to just drop the distro versions

Reply:
I have updated the newer versions. Please check.